### PR TITLE
Activity Timeline blocks

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```
Screenshot 2024-05-21 at 15 38 12
Screenshot 2024-05-21 at 15 38 19
Screenshot 2024-05-21 at 15 38 27
Screenshot 2024-05-21 at 15 38 35
Screenshot 2024-05-21 at 15 38 52
Screenshot 2024-05-21 at 15 38 58

### CODE REVIEW

File: docs/blocks/foundation/ActivityTimeline.story.tsx [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+  Divider,
+  IconButton,
+  Stack,
+  Step,
+  Stepper,
+  Textarea
+} from '../../../src';
+
+import ArrowUpIcon from '../../assets/arrow_up.svg?react';
+import CloseIcon from '../../assets/close.svg?react';
```
Comment (amcdnl) [inline docs/blocks/foundation/ActivityTimeline.story.tsx:15]: Lets inline all the SVGs for easy copy paste

File: src/layout/Stepper/Stepper.tsx [hunk 2]
```diff
@@ -36,57 +54,65 @@ export const Stepper: FC<StepperProps> = ({
   const totalSteps = childrenStepProps.length - 1;
 
   return (
-    <div className={theme.base}>
+    <MotionGroup
+      className={cn(theme.base, className)}
+      initial={animated ? 'initial' : null}
```
Comment (amcdnl) [inline src/layout/Stepper/Stepper.tsx:59]: How does this animate? You don't pass the directions.

File: src/layout/Stepper/Stepper.tsx [hunk 3]
```diff
@@ -1,10 +1,15 @@
-import { StepperTheme } from '@/layout';
+import { MotionGroup, MotionItem, StepperTheme } from '@/layout';
 import { cn, useComponentTheme } from '@/utils';
 import React, { Children, FC, Fragment, PropsWithChildren } from 'react';
 
 import { Step, StepProps } from './Step';
 
 export interface StepperProps extends PropsWithChildren {
+  /**
+   * CSS Classname to applied to the steper
```
Comment (amcdnl) [inline src/layout/Stepper/Stepper.tsx:9]:
```suggestion
   * CSS Classname to applied to the Stepper
```

File: src/layout/Stepper/Stepper.tsx (hunk 2 above)
Comment (evgenoid) [reply src/layout/Stepper/Stepper.tsx:59]: This is for animation disabling optionally

File: src/layout/Stepper/Stepper.tsx [hunk 4]
```diff
@@ -19,12 +24,25 @@ export interface StepperProps extends PropsWithChildren {
...
+
+  /**
+   * Display link after last step
+   */
+  continuously?: boolean;
+
+  /**
+   * Animate items appearance
+   */
+  animated?: boolean;
```
Comment (amcdnl) [inline src/layout/Stepper/Stepper.tsx:36]: I've been thinking of allowing passing animated to a lot of our components and exposing the ability to pass the animation instructions. TBD tho. This is fine for now.

**Pull request merged**